### PR TITLE
wt: Added libatomic to system_libs

### PR DESCRIPTION
**Pull request by ashley-b**
Fixed linker error on Raspbian when trying to consume this package

Specify library name and version:  **wt/4.8.0**
Issue #14714

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by toge** [comment]
@ashley-b 
Thank you for your contribution!

To merge your PR, you need to request for Access List.
Please refer https://github.com/conan-io/conan-center-index/issues/4, and write some comment.

File: recipes/wt/all/conanfile.py [hunk 1]
```diff
@@ -185,56 +177,62 @@ def _gather_libs(p):
...
-        return self._cmake
+            tc.variables["CONNECTOR_FCGI"] = self.options.connector_fcgi
+            tc.variables["CONNECTOR_ISAPI"] = False
+        tc.generate()
+
+        cmake_deps = CMakeDeps(self)
+        cmake_deps.generate()
+
+    def layout(self):
+        cmake_layout(self, src_folder="source_folder")
```
Comment (uilianries) [inline recipes/wt/all/conanfile.py:220]:
```suggestion
        cmake_layout(self, src_folder="src")
```

**Review by uilianries — DISMISSED** [review]
LGTM

File: recipes/wt/all/conanfile.py (hunk 1 above)
Comment (ashley-b) [reply recipes/wt/all/conanfile.py:220]: Question: Is it preferred to specifying the src_folder or leave it out as a default argument?
Ive noticed most conan recipes leave it out eg 
```
cmake_layout(self)
```

**Pull request closed without merging**